auto_dev/data/repo/templates/autonomy/install.sh

```diff
@@ -156,11 +169,13 @@ function set_env_file () {
 function setup_autonomy() {
     echo "Setting up autonomy"
     echo 'Initializing the author and remote for aea and syncing packages...'
-    author=$(cat ~/.aea/cli_config.yaml | yq -r '.author') || author="ci"
-    poetry run aea init --remote --author $author > /dev/null || exit 1
-    echo 'Done initializing the author and remote for aea using the author: ' $author
-    echo 'To change the author, run the command;
-    `poetry run aea init --remote --author <author>`'
+
+    # Extract author from config file with fallback to ci
+    author="$(grep "^author:" ~/.aea/cli_config.yaml 2>/dev/null | sed 's/author:[[:space:]]*//')" || author="ci"
```

Comment:
so did the old version not work at all or what? 

As it was working on ubuntu and on mac, ngl would rather understand why fedora has this issue than change so much?

Reply:
you were using `yq` to extract the author name which I don't have installed

Reply:
great find :)